default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	//  - auto: The editor will use platform APIs to detect when a Screen Reader is attached.
+	//  - on: The editor will be permanently optimized for usage with a Screen Reader. Word wrapping will be disabled.
+	//  - off: The editor will never be optimized for usage with a Screen Reader.
+	"editor.accessibilitySupport": "auto",
+
+	// Controls whether the editor should automatically close brackets after the user adds an opening bracket.
+	//  - always
+	//  - languageDefined: Use language configurations to determine when to autoclose brackets.
+	//  - beforeWhitespace: Autoclose brackets only when the cursor is to the left of whitespace.
+	//  - never
+	"editor.autoClosingBrackets": "languageDefined",
+
+	// Controls whether the editor should remove adjacent closing quotes or brackets when deleting.
+	//  - always
+	//  - auto: Remove adjacent closing quotes or brackets only if they were automatically inserted.
+	//  - never
+	"editor.autoClosingDelete": "auto",
+
+	// Controls whether the editor should type over closing quotes or brackets.
+	//  - always
+	//  - auto: Type over closing quotes or brackets only if they were automatically inserted.
+	//  - never
+	"editor.autoClosingOvertype": "auto",
+
+	// Controls whether the editor should automatically close quotes after the user adds an opening quote.
+	//  - always
+	//  - languageDefined: Use language configurations to determine when to autoclose quotes.
+	//  - beforeWhitespace: Autoclose quotes only when the cursor is to the left of whitespace.
+	//  - never
+	"editor.autoClosingQuotes": "languageDefined",
+
+	// Controls whether the editor should automatically adjust the indentation when users type, paste, move or indent lines.
+	//  - none: The editor will not insert indentation automatically.
+	//  - keep: The editor will keep the current line's indentation.
+	//  - brackets: The editor will keep the current line's indentation and honor language defined brackets.
+	//  - advanced: The editor will keep the current line's indentation, honor language defined brackets and invoke special onEnterRules defined by languages.
+	//  - full: The editor will keep the current line's indentation, honor language defined brackets, invoke special onEnterRules defined by languages, and honor indentationRules defined by languages.
+	"editor.autoIndent": "full",
+
+	// Controls whether the editor should automatically surround selections when typing quotes or brackets.
+	//  - languageDefined: Use language configurations to determine when to automatically surround selections.
+	//  - quotes: Surround with quotes but not brackets.
+	//  - brackets: Surround with brackets but not quotes.
+	//  - never
+	"editor.autoSurround": "languageDefined",
+
+	// Controls whether bracket pair colorization is enabled or not. Use `workbench.colorCustomizations` to override the bracket highlight colors.
+	"editor.bracketPairColorization.enabled": false,
```

Comment:
I think this should be `true`.
![Uploading Screen Shot 2022-04-22 at 2.49.15 PM.png…]()

Reply:
Woah, this is a default setting? I installed a third-party extension to get this behavior 😂